app/styles/debug.module.css

```diff
@@ -0,0 +1,235 @@
+  padding: 1.5rem 2rem;
+  border-radius: 10px;
+  box-shadow: var(--color-grayish-blue-shadow) 1px 1px 4px 0;
+  transition: all 0.2s;
+}
+
+.debug__grid-item:hover {
+  box-shadow: var(--color-grayish-blue-shadow) 6px 6px 4px 0;
+}
+
+.debug-profile {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  color: var(--color-navyblue);
+}
+
+.debug-profile__name {
+  font-size: 2rem;
+  text-align: center;
+}
+
+.debug-profile__image-container {
+  border-radius: 50%;
+  width: 10rem;
+  height: 10rem;
+  overflow: hidden;
+}
+
+.debug-profile__image-container img {
+  width: 100%;
+  object-fit: cover;
+}
+
+.debug-social ul,
+.debug-user ul {
+  height: 100%;
+  display: flex;
+  gap: 1.2rem;
+  flex-direction: column;
+  justify-content: center;
+  word-break: break-all;
+}
+
+.debug-user__roles-container {
+  display: flex;
+  flex-direction: row;
+  max-height: fit-content;
+  gap: 1rem;
+}
+
+.debug-user__roles-container ul {
+  margin-top: 0.08rem;
+  gap: 1rem;
+}
+
+.debug-features {
+  background: var(--color-navyblue);
+  color: var(--color-white);
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+}
+
+.debug-features__feature-flag {
+  margin-bottom: 3rem;
+}
+
+.debug-features__feature-flag ul {
+  margin-top: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.debug-features__superuser {
+  margin-top: auto;
+}
+
+.debug-features__superuser__indicator-container {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+}
+
+.debug-features__superuser__indicator {
+  border-radius: 50%;
+  width: 1.2rem;
+  height: 1.2rem;
+  background: var(--color-shade-grey);
+}
+
+.debug-features__superuser__indicator--active {
+  background: var(--color-green);
+}
+
+.debug-features__superuser__message {
+  margin-top: 0.6rem;
+  font-size: 0.9rem;
+}
+
+.debug-features__superuser-privileges {
+  display: flex;
+  gap: 1rem;
+}
+
+.privileges-toggle {
+  position: relative;
+  display: inline-block;
+  width: 3rem;
+  height: 1.6rem;
+  max-height: 3rem;
+}
+
+.privileges-toggle__checkbox {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+.privileges-toggle__slider {
+  position: absolute;
+  cursor: pointer;
+  inset: 0;
+  background-color: var(--color-shade-grey);
+  border-radius: 2rem;
+  transition: 0.4s;
+}
+
+.privileges-toggle__slider::before {
+  position: absolute;
+  content: "";
+  height: 1.2rem;
+  width: 1.2rem;
+  left: 0.25rem;
+  bottom: 0.2rem;
+  border-radius: 50%;
+  background-color: var(--color-white);
+  transition: 0.4s;
+}
+
+.privileges-toggle__checkbox:checked + .privileges-toggle__slider {
+  background-color: var(--color-blue);
+}
+
+.privileges-toggle__checkbox:checked + .privileges-toggle__slider::before {
+  transform: translateX(1.35rem);
+}
+
+.debug__error-message {
+  text-align: center;
+}
+
+.debug__error-message p {
+  margin-bottom: 1rem;
+}
+
+@media only screen and (width > 1600px) {
```

Comment:
are we using this breakpoint?

Reply:
No, this big breakpoint isn't used. I used this to keep the design well for large screens too 
![Screenshot_15](https://github.com/Real-Dev-Squad/website-www/assets/87164140/d2a94745-168b-4ad4-9393-7ae9c998e7d6)

Reply:
Updated